examples/ex18p.cpp

```diff
@@ -0,0 +1,328 @@
+//                       MFEM Example 18 - Parallel Version
```

Comment:
It will be best if we add the `--strumpack` option to `ex11p` instead of having a separate example. Can we do that?